component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -662,7 +662,9 @@ public static void redirectToEnableTOTPReqPage(HttpServletRequest request, HttpS
         if (isEnrolUserInAuthenticationFlowEnabled(context, runtimeParams)) {
             String multiOptionURI = getMultiOptionURIQueryParam(request);
             String queryParams = "sessionDataKey=" + context.getContextIdentifier() + "&authenticators=" +
-                    TOTPAuthenticatorConstants.AUTHENTICATOR_NAME + "&type=totp" + "&ske=" + skey + multiOptionURI;
+                    TOTPAuthenticatorConstants.AUTHENTICATOR_NAME + "&type=totp" + "&ske=" + skey +
+                    "&t=" + context.getLoginTenantDomain() + "&sp=" +
+                    Encode.forUriComponent(context.getServiceProviderName()) + multiOptionURI;
```

Comment:
In previously `skey` values was build with `"&ske=" + skey + multiOptionURI`. New changes only have `"&ske=" + skey`. What is values passing through `multiOptionURI`. Is it a separate query param.

Reply:
Yes it's a seperate param with the format `&multiOptionURI=<VALUE>`